### DISCUSSION TITLE
Storing quotes for ordering a part

### DISCUSSION THREAD

**Discussion by temmeand** [inventree/InvenTree#4480] [Q&A] [answered]
Any thoughts on the "best" place to store a quote for an item?

I am trying to attach a suppliers quote for an item to document the price I enter for the supplier part. I do not see a place for this. Is there a place in the current design intended for this? If not, any suggestions on what might be an appropriate place? Right now, I'm thinking of attaching it to the base part. 

When preparing bids for items, we often have to include justification for prices. A future workflow would be to export a bid that includes the full BOM with costs and any included justifications.

**Comment by miggland** [comment, marked as answer]
Supplier parts can't handle attachments at the moment, but manufacturer parts can.

Soon, companies will support attachments - probably version 0.11.0 (https://github.com/inventree/InvenTree/pull/4346)

I usually attach quotes to POs, because then there is a connection between supplier, parts, amounts, and so on, which normally (in my case) play a role with prices and conditions.

**Comment by SchrodingersGat** [comment]
The suggestion by @miggland here is the best approach given current capabilities of the software
